documentation-site/components/gallery.js

```diff
@@ -0,0 +1,354 @@
+      href: '/components/spinner',
+      Component: thumbnails.SvgSpinner,
+    },
+    {
+      label: 'Toast',
+      href: '/components/toast',
+      Component: thumbnails.SvgToast,
+    },
+  ],
+  Surfaces: [
+    {
+      label: 'card',
+      href: '/components/card',
+      Component: thumbnails.SvgCard,
+    },
+    {
+      label: 'Drawer',
+      href: '/components/drawer',
+      Component: thumbnails.SvgDrawer,
+    },
+    {
+      label: 'Modal',
+      href: '/components/modal',
+      Component: thumbnails.SvgModal,
+    },
+    {
+      label: 'Pop-Over',
+      href: '/components/popover',
+      Component: thumbnails.SvgPopover,
+    },
+    {
+      label: 'Tool-Tip',
+      href: '/components/tooltip',
+      Component: thumbnails.SvgTooltip,
+    },
+  ],
+  Utility: [
+    {
+      label: 'Aspect Ratio Box',
+      href: '/components/aspect-ratio-box',
+      Component: thumbnails.SvgAspectRatioBox,
+    },
+    {
+      label: 'Flex Grid',
+      href: '/components/flex-grid',
+      Component: thumbnails.SvgFlexGrid,
+    },
+    {
+      label: 'Layer',
+      href: '/components/layer',
+      Component: thumbnails.SvgLayer,
+    },
+    {
+      label: 'A11y Validator',
+      href: '/components/unstable-a11y',
+      Component: thumbnails.SvgUnstableA11y,
+    },
+    {
+      label: 'Tokens',
+      href: '/components/tokens',
+      Component: thumbnails.SvgTokens,
+    },
+    {
+      label: 'Block',
+      href: '/components/block',
+      Component: thumbnails.SvgBlock,
+    },
+  ],
+};
+
+function Thumbnail({children, label, href, scale = 1}) {
+  const [css, theme] = useStyletron();
+  return (
+    <a
+      href={href}
+      className={css({
+        display: 'flex',
+        flexDirection: 'column',
+        border: `solid 1px ${theme.colors.borderAlt}`,
+        marginRight: theme.sizing.scale800,
+        marginBottom: theme.sizing.scale800,
+        textDecoration: 'none',
+        transitionProperty: 'all',
+        transitionDuration: theme.animation.timing100,
+        transitionTimingFunction: theme.animation.easeInOutCurve,
+        ':hover': {
+          border: `solid 1px ${theme.colors.borderFocus}`,
+        },
+      })}
+    >
+      <div
+        className={css({
+          fontFamily:
+            'SFMono-Regular, Consolas, "Liberation Mono", Menlo, Courier, monospace',
+          fontSize: '12px',
+          color: `${theme.colors.foregroundAlt}`,
+          paddingTop: '8px',
+          paddingLeft: '12px',
+          paddingRight: '0',
+          paddingBottom: '8px',
+          borderBottom: `solid 1px ${theme.colors.borderAlt}`,
+        })}
+      >
+        {'baseui/' + href.split('/')[2]}
+      </div>
+      <div
+        className={css({
+          paddingTop: '12px',
+          paddingLeft: '12px',
+          paddingRight: '12px',
+          paddingBottom: '12px',
+        })}
+      >
+        {children}
+      </div>
+    </a>
+  );
+}
+
+function Section({nature}) {
```

Comment:
should we rename `nature` to `category`? I think that's how we refer to these in other parts of the code base

Reply:
not sure why i called this `nature`- sounds more mysterious maybe